GeometryCore/src/Polygon.cpp

```diff
@@ -252,4 +252,83 @@ namespace geom_utils
         return poly;
     }
 
+    inline FPoint2D vecRot90CW(const FPoint2D& point) 
```

Comment:
BTW, good candidate to be moved to LinearAlgebra. It's common function and can be used widely somewhere else